query-service-impl/src/test/java/org/hypertrace/core/query/service/pinot/ExecutionContextTest.java

```diff
@@ -160,10 +160,12 @@ public void testReferencedColumns() {
             .setOperator(Operator.EQ);
     Filter startTimeFilter =
         QueryRequestBuilderUtils.createTimeFilter(
-            "Trace.start_time_millis", Operator.GT, currentTimeInMillis - 1000 * 60 * 60 * 24);
+            "Trace.start_time_millis",
+            Operator.GT,
+            startTimeInMillis - Duration.ofHours(24).toMillis());
```

Comment:
you're subtracting 24 hours off of one hour past the epoch - that's going to give you a negative time. Instead, use start time directly and add 24h to get your end time.